code/modules/events/spacevine.dm

```diff
@@ -135,8 +137,8 @@
 	quality = MINOR_NEGATIVE
 
 /datum/spacevine_mutation/vine_eating/on_spread(obj/structure/spacevine/holder, turf/target)
-	var/obj/structure/spacevine/prey = locate() in target
-	if(prey && !prey.mutations.Find(src))  //Eat all vines that are not of the same origin
+	var/obj/structure/spacevine/prey
+	for(prey in target) // Destroys any vine on target tile. The checks for if this should be done are in the spread() proc.
```

Comment:
move the var definition into the for here, since you don't care about it afterwards.

Reply:
Done.